Easydict/Swift/Service/Caiyun/CaiyunTranslateType.swift

```diff
@@ -13,38 +13,43 @@ struct CaiyunTranslateType: RawRepresentable {
 
     // Align with the web interface https://fanyi.caiyunapp.com/#/
     static let supportedTypes: [Language: [Language]] = [
-        .simplifiedChinese: [.english, .japanese, .korean, .spanish, .french, .russian],
-        .english: [.simplifiedChinese, .spanish, .french, .russian],
-        .japanese: [.simplifiedChinese],
-        .korean: [.simplifiedChinese],
-        .spanish: [.simplifiedChinese, .english, .french, .russian],
-        .french: [.simplifiedChinese, .english, .spanish, .russian],
-        .russian: [.simplifiedChinese, .english, .spanish, .french],
+        .simplifiedChinese: [.traditionalChinese, .english, .japanese, .korean, .spanish, .french, .russian],
```

Comment:
Thanks for the PR.

Is it possible to add the new supported languages as target languages to `.simplifiedChinese` and `.traditionalChineses`?

From reading the docs, new languages like Vietnamese seems to be supported as target languages for Chinese.

I would also try testing the new languages as target languages for English and other source langauges.

<img width="472" alt="image" src="https://github.com/user-attachments/assets/dba3860b-fadc-484e-a024-5f2c6f84f1bd" />

<img width="801" alt="image" src="https://github.com/user-attachments/assets/e10342c9-2b4f-4790-b928-95a209e58ca3" />

Reply:
Thank you for your review. I have updated the translation direction based on the document. PTAL.